### PR TITLE
Add class signature to output of `find_formula()`

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
Erf I had to changed all the tests from expect_equal to expect_equivalent. Opening a new branch since I'm not sure that I didn't break anything.

The purpose of this class signature is to be able to do further stuff with formulas, like nice formatting, pretty printing etc.

### CODE REVIEW

**Comment by strengejacke** [comment]
No, you don't need to... ignore_attr is enough (assuming all your packages are up to date)

**Comment by strengejacke** [comment]
Furthermore, `expect_equivalent` is deprecated.

**Comment by DominiqueMakowski** [comment]
The tests failed even with it with expect_equal

**Comment by DominiqueMakowski** [comment]
> Furthermore, expect_equivalent is deprecated.

Did not know that

**Comment by strengejacke** [comment]
image

I don't see any failing tests, nor do I have problems locally.

**Comment by strengejacke** [comment]
<sup>(sorry for being so fast with closing this, but it took hours to update the test files and make the changes from `expect_equivalent` to `expect_equal`) ;-)</sup>

**Comment by DominiqueMakowski** [comment]
> (sorry for being so fast with closing this, but it took hours 

It just took some time as well, no but you missed that I did not only changed the tests, but also the find_formula function which is the core of this pr

**Comment by DominiqueMakowski** [comment]
And it's the changes to find_formula that made the tests fail @strengejacke

**Comment by strengejacke** [comment]
Which change?

File: R/find_formula.R [hunk 1]
```diff
@@ -1431,3 +1474,15 @@ find_formula.BFBayesFactor <- function(x, ...) {
   }
   f
 }
+
+
+
+# Helpers and Methods -----------------------------------------------------
+
+
+.find_formula_return <- function(f){
```
Comment (DominiqueMakowski) [inline R/find_formula.R:1483]: That one @strengejacke

**Comment by strengejacke** [comment]
I can't find such a function in https://github.com/easystats/insight/blob/master/R/find_formula.R?

**Comment by strengejacke** [comment]
image

**Comment by DominiqueMakowski** [comment]
You mean that small internal that adds classes? Of course you don't see it on master since you didn't merge my PR ^^

**Comment by strengejacke** [comment]
Ah, got it! I thought your PR was just about changing all tests...

**Comment by DominiqueMakowski** [comment]
Would you be my angel and re-add the class signature whenever you can while I go to sleep 😴? Thanks 🙏😁

**Pull request closed without merging**